packages/commons/src/fhir/extension.ts

```diff
@@ -258,6 +258,6 @@ export function findExtension<URL extends AllExtensions['url']>(
 ): Extract<AllExtensions, { url: URL }> | undefined {
   return listOfExtensions.find(
     (extension): extension is Extract<AllExtensions, { url: URL }> =>
-      extension.url === url
+      extension && extension.url === url
```

Comment:
Is there a specific valid case when the extension can be null? As per the type signature of this function, there really should be a case like that. If there is such a case, let's rather ensure null never gets into a list of extensions.